WeTransferPRLinter/Sources/WeTransferPRLinter/WeTransferPRLinter.swift

```diff
@@ -72,6 +72,8 @@ public enum WeTransferPRLinter {
 
                 return true
             }
+        } catch let error as LocationError where error.isMissingError {
```

Comment:
Nit: would it make sense to do `error.reason == .missing`. For example, it would enable someone who reads the code to lookup documentation by clicking on the `.missing` part.

Reply:
Yeah, I started with that but `LocationErrorReason` does not conform to `Equatable` which is why I went for this route. Easier to maintain since we don't own the library including `LocationErrorReason`.